index.js

```diff
@@ -20,4 +24,113 @@ navLinks.forEach((link) => {
     navList.classList.remove('nav-active');
     hamburger.classList.remove('toggle');
   });
-});
\ No newline at end of file
+});
+
+const projectArrays = [
+  {
+    name: 'Mobile post stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry"s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    technologies: ['html', 'Bootstrap', 'Ruby'],
+    id: '002',
+  },
+  {
+    name: 'Mobile post stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry"s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    technologies: ['html', 'Bootstrap', 'Ruby'],
+    id: '003',
+  },
+  {
+    name: 'Mobile post stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry"s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    technologies: ['html', 'Bootstrap', 'Ruby'],
+    id: '004',
+  },
+  {
+    name: 'Mobile post stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry"s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    technologies: ['html', 'Bootstrap', 'Ruby'],
+    id: '001',
+  },
+  {
+    name: 'Mobile post stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry"s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    technologies: ['html', 'Bootstrap', 'Ruby'],
+    id: '005',
+  },
+  {
+    name: 'Mobile post stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry"s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    technologies: ['html', 'Bootstrap', 'Ruby'],
+    id: '006',
+  },
+];
+
+const createProjectElement = (project) => {
+  const projectcontainer = document.createElement('li');
+  projectcontainer.classList.add('project-container');
+  projectcontainer.innerHTML = `
```

Comment:
Tip for future projects:
`innerHTML`  can be harmful when you don't have control over the data being injected. It is safer to use textContent. 
You can use `innerHTML` for this project since you have control over your data but it is not recommended practice

Reply:
Thanks for the Tip, 👍  I will make sure to implement it in future projects